tofu/imas2tofu/_core.py

```diff
@@ -2802,6 +2862,77 @@ def to_Plasma2D(self, tlim=None, dsig=None, t0=None,
                     description_2d=None,
                     plot=None, plot_sig=None, plot_X=None,
                     bck=True, dextra=None, nan=True, pos=None, shapeRZ=None):
+        """ Export the content of some ids as a tofu Plasma2D object
+
+        Some ids typically contain plasma 1d (radial) or 2d (mesh) profiles
+        They include for example ids:
+            - core_profiles
+            - core_sources
+            - edge_profiles
+            - edge_sources
+            - equilibrium
+
+        tofu offers a class for handling multiple profiles characterizing a
+        plasma, it's called Plasma2D
+        This method automatically identifies the ids that may contain profiles,
+        extract all profiles (i.e.: all profiles identified by a shortcut, see
+        self.get_shortcuts()) and export everything to a fresh Plasma2D
+        instance.
+
+        Parameters
+        ----------
+        tlim:   None / list
+            Restrict the loaded data to a time interval with tlim
+            if None, loads all time steps
+        dsig:   None / dict
+            Specify exactly which data (shortcut) should be loaded by ids
+            If None, loads all available data
+        t0:     None / float / str
+            Specify a time to be used as origin:
+                - None: absolute time vectors are untouched
+                - float : the roigin of all time vectors is set to t0
+                - str : the origin is taken from an event in ids pulse_schedule
+        Name:   None / str
+            Name to be given to the instance
+            If None, a default Name is built
+        occ:    None / int
+            occurence to be used for loading the data
+        config: None / Config
+            Configuration (i.e.: tokamak geometry) to be used for the instance
+            If None, created from the wall ids with self.to_Config().
+        out:    type
+            class with which the output shall be returned
+                - object :  as a Plasma2D instance
+                - dict:     as a dict
+        description_2d: None / int
+            description_2d indiex to be used if the Config is to be built from
```

Comment:
indiex => index. Same in all the other docs :)

Reply:
Thanks !